perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/TasksManagerBlImpl.java

```diff
@@ -361,85 +270,124 @@ public TaskResult getTaskResultById(int taskResultId) {
 		return resourceStateList;
 	}
 
-	@Override
-	public List<ServiceState> getFacilityServicesState(PerunSession sess, Facility facility) {
-
-		Map<Service, ServiceState> serviceStates = new HashMap<>();
-
-		// fill states for all services which are currently on facility
-		for (Service service : perun.getServicesManagerBl().getAssignedServices(sess, facility)) {
-
-			serviceStates.put(service, new ServiceState(service, facility));
-			serviceStates.get(service).setBlockedOnFacility(getServicesManagerBl().isServiceBlockedOnFacility(service, facility));
+	public ServicesManagerBl getServicesManagerBl() {
```

Comment:
The constructor should be the first in the file before other methods.

Reply:
And it is, I do not see other methods before constructor.

Reply:
My bad. I meant this getter to move right after constructor if possible.